src/v/security/tests/role_store_bench.cc

```diff
@@ -0,0 +1,144 @@
+/*
+ * Copyright 2024 Redpanda Data, Inc.
+ *
+ * Licensed as a Redpanda Enterprise file under the Redpanda Community
+ * License (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * https://github.com/redpanda-data/redpanda/blob/master/licenses/rcl.md
+ */
+
+#include "random/generators.h"
+#include "security/role.h"
+#include "security/role_store.h"
+
+#include <seastar/testing/perf_tests.hh>
+#include <seastar/util/later.hh>
+
+#include <boost/range/irange.hpp>
+
+#include <random>
+#include <utility>
+#include <vector>
+
+using namespace security;
```

Comment:
remove `using namespace`, put using defs and static data in anon namespace instead